fish-rust/src/builtins/type.rs

```diff
@@ -119,15 +119,13 @@ pub fn r#type(
                     }
                     if opts.path {
                         if let Some(orig_path) = props.copy_definition_file() {
-                            streams.out.append(orig_path);
+                            streams.out.appendln(orig_path);
                         } else {
-                            streams.out.append(path);
+                            streams.out.appendln(path);
                         }
-                        streams.out.append1('\n');
                     } else if !opts.short_output {
                         streams.out.append(wgettext_fmt!("%ls is a function", arg));
-                        streams.out.append(wgettext_fmt!(" with definition"));
-                        streams.out.append1('\n');
+                        streams.out.appendln(wgettext!(" with definition"));
```

Comment:
I find this splitting of `wgettext` really weird, there should be some guidelines against doing that, same with including the trailing `\n` in the text to be translated and properly separating punctuation-differences (as we observed in the builtins/string rewrite)